queries.sql

```diff
@@ -1,3 +1,9 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name, date_of_birth FROM animals WHERE name IN('Agumon', 'Pikachu');
+SELECT name, escape_attempts From animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = true;
+SELECT * FROM animals WHERE name != 'Gabumon';
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
```

Comment:
- [ ] according to the requirements you should also include a query for the following question:
  - list the name of all animals that are neutered and have less than 3 escape attempts.

Reply:
Hi, @mikethreels!

Thank you for your comment! I worked on that suggestion, and it is already solved!